src/utils/Loading.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+const Loading: React.FC = () => (
```

Comment:
I wouldn't copy in the loader from console, and it's a bad practice for plugins to reference console styles directly since they might change.

We should look at switching console to use the PatternFly loader, and I'd use that here in the plugin (even if different from console today).